files/benchmark_job.yml

```diff
@@ -124,10 +128,26 @@
                                 compiler="http://10.40.0.13/toolchains/${file}"
                         fi
 
+                        if [ -d "${WORKSPACE}/mr-provisioner-client" ]; then
+                            rm -rf "${WORKSPACE}/mr-provisioner-client"
+                        fi
+                        git clone https://github.com/Linaro/mr-provisioner-client.git "${WORKSPACE}/mr-provisioner-client"
+
+                        mr_provisioner_url="http://10.40.0.11:5000"
+                        mr_provisioner_token=$(cat "/home/${NODE_NAME}/mrp_token")
+                        mr_provisioner_machine_name="${node_type}bench"
+
+                        machine_ip=$( "${WORKSPACE}/mr-provisioner-client/mrp_client.py" getip ${mr_provisioner_machine_name} --mrp-url=${mr_provisioner_url} --mrp-token=${mr_provisioner_token})
+
+                        cat << EOF > "${WORKSPACE}/hosts"
+                        [target]
+                        ${machine_ip}
+                        EOF
+
                         cat << EOF > benchmark_job.yml
-                        mr_provisioner_url: http://10.40.0.11:5000
-                        mr_provisioner_token: $(cat "/home/${NODE_NAME}/mrp_token")
-                        mr_provisioner_machine_name: ${node_type}bench
+                        mr_provisioner_url: ${mr_provisioner_url}
```

Comment:
Why do we need these parameters? We're not using MrP at all in the Ansible part.